### ISSUE TITLE
throw Error for ISO search

### ISSUE THREAD

**Issue by triori** [richorama/country-code-lookup#39]
Hi there,
My point if you return null in other case, why you throw new Error in ISO case?
`throw new Error('cannot determine ISO code type')` in ISO case
VS
`return null` for other cases.
I think it is unexpectable result, isn't it?

**Comment by richorama** [comment]
I agree that there is some inconsistency here, but the error indicates that the input format is possibly wrong, the other indicates that there isn't a match (another way for the input being wrong I suppose).

Happy to change this behaviour with an update to the major version number to represent a change in the API

**Comment by jjsimps** [comment]
At minimum I think that the behavior should be documented. Especially if you stick to throwing an Error.
